web/src/app/listing/component.ts

```diff
@@ -75,7 +75,41 @@ export class ListingComponent implements OnInit{
   }
 
   public get tooltipDescription (): string {
-    return this.listing.description || 'No description available :('; 
+    return this.listing.description || 'No description available :(';
+  }
+
+  public getSectionClosedStatus (): string {
```

Comment:
Avoid using "magic strings" as return values from functions. We can refactor this to make it cleaner. I suggest: moving the counting logic to its own function `getClosedSectionCount`, and comparing the closed count to the number of listings in a separate function, or even in the view. Then you won't need the `"allClosed"` and `"someClosed"` strings. The same goes for conflicts.

Reply:
Fixed! @Bad-Science